servicemigration/logconfig.py

```diff
@@ -23,10 +22,11 @@ def deserialize(data):
     for d in data:
         lc = LogConfig()
         lc._LogConfig__data = d
-        lc.path = d.get("path", d.get("Path"))
-        lc.logType = d.get("type", d.get("Type"))
-        lc.filters = d.get("filters", d.get("Filters"))
-        lc.logTags = logtag.deserialize(d.get("LogTags", []))
+        lc.path = d.get("Path", default["Path"])
```

Comment:
If a service has lowercase attributes for path, type, or filters, won't this ignore those values and replace them with defaults? Since the serializer used to produce lowercase keys, I know there's going to be cases of services with these fields in lowercase.